docs/discriminators.md

```diff
@@ -15,7 +15,17 @@ key (defaults to the model name). It returns a model whose schema
 is the union of the base schema and the discriminator schema.
 
 ```javascript
-[require:The `model.discriminator()` function]
+const db = mongoose.createConnection('mongodb://localhost:27017/mongoose_test');
```

Comment:
Why is this copy/pasted in here? Acquit should be pulling this code from the discriminators.test.js file.

Reply:
I tried to get that section in using acquit but it wouldn't do it.

```
beforeEach('Test', function(done) {
...
});
```
in discriminators.md

```javascript
[require:Test]
```

Reply:
Then you need to figure out why the example is not getting pulled in :)